### ISSUE TITLE
Can EDScout run on a different computer with LAN access to journals?

### ISSUE THREAD

**Issue by bbourgoin** [joncage/ed-scout#163]
This is a question. I didn't know where else to ask it. I'd like to run EDScout on a computer that isn't the ED computer. I already have the journals folder mapped as a drive on the 2nd computer. I took a look through the 1 or 2 files that contained config info and I didn't see a pointer to the journals folder location.

As a hard core explorer, I absolutely love this software. I have a dedicated 8 inch screen for it. Thank you for your work on this.

Bruno

**Comment by joncage** [comment]
Any reason you can't run EDScout on the same machine as Elite and remotely connect a browser to it? That's the preferred way to do this I'd suggest.

See the command line options here: https://github.com/joncage/ed-scout#command-line-options

**Comment by bbourgoin** [comment]
Hi! Thanks for the quick answer.

I was hoping to move as many third party apps to a separate dedicated computer with lan links to the journal files as possible. This leaves the main computer running ED and any virtual keyboard tools like Gameglass. Mostly just trying to reduce load on the ED computer giving it more memory and CPU cycles for ED.

I realize EDSCOUT is pretty small and I don't think it's a memory or CPU hog, so if it stays on the main computer it isn't a big deal. I was just wondering if that capability existed. 

Thanks for this amazing software. I Don't fly without it.

Bruno


On February 17, 2021 9:19:52 AM EST, joncage <notifications@github.com> wrote:
>Any reason you can't run EDScout on the same machine as Elite and
>remotely connect a browser to it? That's the preferred way to do this
>I'd suggest.
>
>See the command line options here:
>https://github.com/joncage/ed-scout#command-line-options
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/joncage/ed-scout/issues/163#issuecomment-780587456

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by joncage** [comment]
Yeah, could be done to make it a config or command line option. You'd probably need to use the -polling command line option as well as I doubt it would get the normal filesystem triggers to indicat the files have changed while operating over the lan.
